src/programs/aws.ts

```diff
@@ -186,6 +186,7 @@ export const AWSProgram = (opts: OptionValues) => {
           };
 
     /* eslint-disable no-new */
+    const CONTAINER_SERVICE_URL = containerService.url.endsWith("/") ? containerService.url.slice(0, -1) : containerService.url
```

Comment:
Can you move this up above the `eslint-disable no-new`? That line should be associated with the following line, so this should be moved up above.

Reply:
Ah yes, sorry, didn't notice that or have the pre-commit running.  Should be good now